### DISCUSSION TITLE
RQ 3 viz

### DISCUSSION THREAD

**Discussion by nickdgardner** [ohtap/ohtap#28] [General]
[placeholder question]

**Comment by Yibing-Du** [comment]
@ebf77 @kmcdono2 

Hi Estelle and Katie, here's a series of visualization for RQ3. These are based on currently available metadata and NVivo outputs. I'm playing with different ways of presenting them (raw values vs. categories; histogram, bar chart, scatterplot with sizes representing number of instances that lie at a certain position), so many would essentially contain the same information. These are made with R since I find its visualization much easier, but once we settle down at what's needed, I can switch to Python. 

The following files are named to reflect its content. The first 4 histograms + 2 pie charts are by collection, whereas the remaining scatterplots are by individual. The next step would be presenting the intersectionality (e.g. Black American born in 1940s or college grads interviewed in their 80s), which I don't have a great idea about. 

========================by collection========================

![histogram_category_extent_by_collection_2-1](https://user-images.githubusercontent.com/27621580/106566356-1060cd80-64e5-11eb-9829-520ac5d5ff15.png)
![histogram_category_length_by_collection_2-1](https://user-images.githubusercontent.com/27621580/106566361-10f96400-64e5-11eb-867e-1ac55a2f4cf9.png)
![histogram_raw_extents_by_collection_2-1](https://user-images.githubusercontent.com/27621580/106566363-1191fa80-64e5-11eb-9635-cfe65aae3b62.png)
![histogram_raw_length_by_collection_2-1](https://user-images.githubusercontent.com/27621580/106566365-122a9100-64e5-11eb-860b-9ee2ffb5c100.png)
![pie_category_extent_by_collection_2-1](https://user-images.githubusercontent.com/27621580/106566366-122a9100-64e5-11eb-85f8-050292572f12.png)
![pie_category_length_by_collection_2-1](https://user-images.githubusercontent.com/27621580/106566369-12c32780-64e5-11eb-8697-308106c939cc.png)

========================by individual (age)========================

![scatter_size_agecategory_category_extent_2-1](https://user-images.githubusercontent.com/27621580/106566469-34bcaa00-64e5-11eb-9a83-7d241d535134.png)
![scatter_size_agecategory_category_length_2-1](https://user-images.githubusercontent.com/27621580/106566473-35554080-64e5-11eb-9fbd-9fd9d5632901.png)
![scatter_size_agecategory_raw_extent_2-1](https://user-images.githubusercontent.com/27621580/106566474-35edd700-64e5-11eb-8f92-c1efbe76a8af.png)

========================by individual (birth decade)========================

![scatter_size_birthdecade_category_extent_2-1](https://user-images.githubusercontent.com/27621580/106566534-47cf7a00-64e5-11eb-8505-3964859b552c.png)
![scatter_size_birthdecade_category_length_2-1](https://user-images.githubusercontent.com/27621580/106566538-48681080-64e5-11eb-9bff-03bb9117992c.png)
![scatter_size_birthdecade_raw_extent_2-1](https://user-images.githubusercontent.com/27621580/106566543-4900a700-64e5-11eb-984c-6b650600d200.png)

========================by individual (education)========================

![scatter_size_education_category_length_2-1](https://user-images.githubusercontent.com/27621580/106566576-5453d280-64e5-11eb-994a-f7f0991568df.png)
![scatter_size_education_category_extent_2-1](https://user-images.githubusercontent.com/27621580/106566572-53bb3c00-64e5-11eb-8843-6f34d4197b03.png)

========================by individual (race)========================

![scatter_size_race_category_extent_2-1](https://user-images.githubusercontent.com/27621580/106566601-5c137700-64e5-11eb-9e4e-ee6f8f1e4f22.png)
![scatter_size_race_category_length_2-1](https://user-images.githubusercontent.com/27621580/106566603-5cac0d80-64e5-11eb-9f82-1cbc61d10865.png)

**Comment by Yibing-Du** [comment]
Hi everyone! @ebf77 @kmcdono2 @njmarine @nickdgardner 

I've updated on both Google Sheet and Doc and uploaded my code to the RQ3 folder. I cleaned up my old codes and implemented the text analysis methods. I don't have any particular visualization to show here but I think I'll make some exploratory plots for the text analysis methods that Nick proposed before our meeting. 

I know that the visualizations (with metadata) above are probably still confusing. If there's anything we're particularly interested in, I'm happy to work on that in this upcoming week.

**Reply by ebf77** [reply]
Thanks, Yibing.  A few questions: do you have a table with the data that you used for the visualizations? Can you link to that? I'm more interested in the individual than the collection level analysis. Can you provide very precise labels for your x and y axes on the individuals for length of "speech events." I don't know what the "age category" refers to  - age when, as opposed to birth year/decade? Is it age at time of interview? Also, can you fill in the precise labels on the individual interview displays? In https://user-images.githubusercontent.com/27621580/106566543-4900a700-64e5-11eb-984c-6b650600d200.png, for example, y=number of event extents per interview or overall or what? The n on the right size refers to what - number of interviews or number of words? I sense that this is a promising display but I can't figure it out! (is this display title something like: Frequency and Length of Rape Cluster Terms in Interviews across Birth Decades ?

I'll go over protocols for data displays on Saturday but here is a draft preview: 

1 For all displays, include: name of originator, date created (and v.# if tweaked), full labeling of rows/columns and of x and y axes, with explanatory titles. Add link to table or spreadsheet with the data basis for display. I may ask to show the total n for some displays (and for x axes categories such as race. 

2. Calculate by finer degrees but for display consolidate

a. Educational levels:
Up to 12th grade
Up to college degree
Graduate or professional degree
No Information

b. Birth Cohorts
1870-1889
1890-1909
1910-1929
1930-1949
1950-1969
1970-1989
 
Finally, have you gone over these graphs with Nick? 
Let me know if you have any questions this week or have revision to show me before Saturday. 
Gratefully,
Estelle
